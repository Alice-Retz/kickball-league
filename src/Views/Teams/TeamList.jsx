export default function TeamList() {
	return (
		<>
			<ul>
				<li>Team 1</li>
				<li>Team 2</li>
				<li>Team 3</li>
			</ul>
		</>
	);
}
