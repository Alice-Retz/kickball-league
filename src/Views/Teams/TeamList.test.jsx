import { render, screen } from '@testing-library/react';
import { MemoryRouter, Route } from 'react-router-dom';
import TeamList from './TeamList';

it('displays a list of teams', async () => {
	const { container } = render(
		<MemoryRouter initialEntries={['/teams']}>
			<Route path='/teams'>
				<TeamList />
			</Route>
		</MemoryRouter>
	);

	const playerName = await screen.findByText('Big Easy Tigers', {
		exact: false,
	});

	expect(playerName).toBeInTheDocument();
	expect(container).toMatchSnapshot();
});
