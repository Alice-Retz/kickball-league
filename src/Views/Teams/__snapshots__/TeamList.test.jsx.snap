// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays a list of teams 1`] = `
<div>
  <h1>
    Teams:
  </h1>
  <ul>
    <li>
      <a
        href="/teams/2"
      >
        Big Easy Tigers
      </a>
    </li>
    <li>
      <a
        href="/teams/3"
      >
        Cajun Country 'Gators
      </a>
    </li>
    <li>
      <a
        href="/teams/4"
      >
        Red Stick Tigers
      </a>
    </li>
  </ul>
</div>
`;
